hack/agent-image/Containerfile-e2e-v2.local

```diff
@@ -0,0 +1,7 @@
+FROM localhost:5000/flightctl-base:latest
+
```

Comment:
could we add a short comment in all of these container images as to the intent?

Reply:
Good idea, doing.